### ISSUE TITLE
Windows x64 Installation Problem w/ Path Names

### ISSUE THREAD

**Issue by GaryFurash** [Laverna/laverna#866]
Even when trying to extract to the root directory (e.g., C:\Laverna) generates the message "Destination Path Too Long" (node-modules

**Comment by wyldphyre** [comment]
I can confirm this on a 64 bit Windows 10 installation.

**Comment by brodiefairhall** [comment]
Same issue on Win 10 64 bit

**Comment by brodiefairhall** [comment]
Same issue with the 32bit zip for 7.51 and 7.5
Tried to Unzip on a linux machine and it failed as well

**Comment by S-ed** [comment]
No wonder here...
![screenshot 1510924380](https://user-images.githubusercontent.com/3008353/32948995-39810ac4-cbb2-11e7-979e-4b4ba1e984d8.png)

**Comment by S-ed** [comment]
No wonder here...
![screenshot 1510924458](https://user-images.githubusercontent.com/3008353/32949038-5ff5a854-cbb2-11e7-9a5d-a5bef317e457.png)

**Comment by wyldphyre** [comment]
I'd heard that Node was bad at dependancy management, but jeez.

**Comment by S-ed** [comment]
@wyldphyre *npm But yeah. I've even thought about writing a paper how to improve it some time ago. But was afraid all the efforts will be wasted and shelved this idea until better times.

**Comment by EggPenguini** [comment]
Expanded zip to C:\L\     Same problem, path too long.  Win10, 64 bit.

**Comment by JimmyRascon** [comment]
I have solved it by unzipping to C:\L\ and omitting the errors. Then I used this workaround to flat the node_modules folder tree in C:\L\resources\app.
https://scottaddie.com/2015/08/16/npm-vs-windows-max_path-limitation/
After that I renamed and moved the "L" folder wherever I want. It's ugly but it works.

**Comment by S-ed** [comment]
@JimmyRascon The problem of this solution is "omitted" files are not there anymore.
Even if you use some third party app (like Total Commander) that could bypass OS limitation of path length (256 characters), OS won't be able to access them.

Probably, node still will be, but not sure.

**Comment by JimmyRascon** [comment]
@S-ed You are right, the omitted files are not there, they are node dependencies and some of them are duplicated. Using the "flatten-packages" command gets the most updated version of every package, in my case it's working but I'm not 100% sure it will not produce any obscure bug.
Here is my Laverna folder in case it is useful for someone.
https://www.dropbox.com/s/xyqviv8sybd3laf/laverna-0.7.51-win32-x64-mod.zip?dl=0

**Comment by ansib** [comment]
Upgrade to a newer node.js and also the npm version would fix this path problem. (Version 6 is LTS).

**Comment by ParkerD13** [comment]
Got the same problem. Almost all updates are from a year ago. The webapp.zip download is broken. I hope Laverna is still alive.

**Comment by funilrys** [comment]
Hi there,

Unfortunately this issue will never be fixed here.

Please report to https://github.com/Laverna/laverna/issues/971#issuecomment-411423965 for the current state of this project and https://github.com/encryptic-team/encryptic for the future of this project.

Have a nice day/night.

Cheers,
Nissar

**Comment by daed** [comment]
I saw that.  I've been able to skip those files and still run Laverna.  

I just want to point out that Encryptic (and my pre-Encryptic Laverna release) don't appear to have those issues, though there's a handful of regressions in features that I'm working on that might be an issue otherwise depending on your use case.
